packages/explat-client/src/internal/state.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Internal dependencies
+ */
+import type { ExperimentAssignment } from '../types';
+import * as Validations from './validations';
+import * as ExperimentAssignments from './experiment-assignments';
+
+const experimentNameToExperimentAssignment: Record< string, ExperimentAssignment | undefined > = {};
+
+export function storeExperimentAssignment( experimentAssignment: ExperimentAssignment ): void {
```

Comment:
It'd be good to document these functions too (generally: anything that's exported).